addons/account/models/account_journal.py

```diff
@@ -232,20 +227,28 @@ def _compute_default_account_type(self):
                 journal.default_account_type = False
 
     @api.depends('type')
-    def _compute_outbound_payment_method_ids(self):
+    def _compute_inbound_payment_method_line_ids(self):
         for journal in self:
+            pay_method_line_ids_commands = [Command.clear()]
             if journal.type in ('bank', 'cash'):
-                journal.outbound_payment_method_ids = self._default_outbound_payment_methods()
-            else:
-                journal.outbound_payment_method_ids = False
+                default_methods = journal._default_inbound_payment_methods()
+                pay_method_line_ids_commands += [Command.create({
+                    'name': pay_method.name,
+                    'payment_method_id': pay_method.id,
+                }) for pay_method in default_methods]
```

Comment:
You could use `available_payment_method_ids` there. 
`for pay_method in journal.available_payment_method_ids.filtered(lambda x: x.type == 'inbound')`